mutation/src/index.ts

```diff
@@ -76,15 +75,19 @@ const mutation = (<Data, Error>() =>
 
           // If it's reset after the mutation, we don't broadcast any state change.
           if (ditchMutationsUntilRef.current <= mutationStartedAt) {
-            setState({ data, isMutating: false, error: undefined })
+            startTransition(() =>
```

Comment:
we use `React.startTransition` here since `isPending` from `useTransition` will always cause rerender even if `isMutation` is not used.